alembic/env.py

```diff
@@ -0,0 +1,84 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from City_CRUD_API.models import *
```

Comment:
Using wildcard imports (e.g., `from City_CRUD_API.models import *`) can lead to namespace pollution and make it difficult to track where specific classes or functions are coming from. Consider importing only the necessary models explicitly.